src/query/gjk/gjk.rs

```diff
@@ -35,7 +35,7 @@ pub enum GJKResult {
 /// The absolute tolerance used by the GJK algorithm.
 pub fn eps_tol() -> Real {
     let _eps = crate::math::DEFAULT_EPSILON;
-    _eps * 10.0
+    _eps * 10_000.0
 }
```

Comment:
I'm not a huge fan of increasing epsilons arbitrarily, but that improves the example added in this PR.

I think exposing this value would be more interesting, and I'd like to have a test stressing false positive, where a high epsilon results in an incorrect result :thinking: